kodilla-exception/src/main/java/com/kodilla/exception/test/ExceptionHandling.java

```diff
@@ -0,0 +1,18 @@
+package com.kodilla.exception.test;
+
+public class ExceptionHandling  {
+
+    public static void main(String[] args) {
+
+        SecondChallenge challenge = new SecondChallenge();
+
+        try {
+            challenge.probablyIWillThrowException(0.5, 1.5);
+        } catch (Exception e) {
+            System.out.println(e);
```

Comment:
A co to wypisze na ekranie? Sprawdzałeś?

Reply:
Wypisze:

java.lang.Exception: Incorrect data
Thank You

Nie bardzo rozumiem o co chodzi...